llama_stack/distribution/build.py

```diff
@@ -43,8 +43,16 @@ def get_provider_dependencies(
     # Extract providers based on config type
     if isinstance(config, DistributionTemplate):
         providers = config.providers
+        run_configs = config.run_configs
+        additional_pip_packages: list[str] = []
+        if run_configs:
+            for run_config in run_configs.values():
+                run_config_ = run_config.run_config(name="", providers={}, container_image=None)
+                if run_config_.inference_store:
```

Comment:
I think there should be a TODO here at least which says "why this was added and what could be an alternate way to get this done (the internal api / registry change I talked about)